kubernetes/cassandra/statefulset.yaml

```
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: cassandra
+spec:
+  serviceName: cassandra
+  replicas: 3
+  template:
+    spec:
+      terminationGracePeriodSeconds: 500
+      containers:
+      - name: cassandra
+        image: cassandra:latest
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 9042
+          name: cql
+        securityContext:
```

If their is no resource request at all, in case of overloaded cluster, Cassandra would receive (mostly) no CPU at all which is undesirable.

```suggestion
        resources:
          requests:
            cpu: "0.5"
            memory: 512Mi
        securityContext:
```